src/Pages/ExercisesPage/ExercisesPage.js

```diff
@@ -1,12 +1,55 @@
-import SpecialButtonsContainer from '../../Molecules/SpecialButtonsContainer/SpecialButtonsContainer';
+import { useState } from 'react';
 
-import ExerciseCardsContainer from '../../Organisms/ExerciseCardsContainter/ExerciseCardsContainer';
+import Modal from '../../Organisms/Modals/Modal';
+import Backdrop from '../../Organisms/Modals/Backdrop';
 import DOGS from '../../Data/Dummy-data/test-data-dogs';
+import ExerciseCardsContainer from '../../Organisms/ExerciseCardsContainter/ExerciseCardsContainer';
+import SpecialButtonsContainer from '../../Molecules/SpecialButtonsContainer/SpecialButtonsContainer';
+const modalData = {
+  penalty: {
+    about: 'KAra',
+    back: 'Anuluj',
+    title: '?!?!?!',
+    confirmation: 'Siur??',
+    theme: 'yellow',
+  },
+  disqualify: {
+    about: 'Dyskwalifikacja',
+    back: 'Anuluj',
+    title: '!!!!!',
+    confirmation: 'Na pewno?',
+    theme: 'red',
+  },
+};
 
 const ExercisesPage = () => {
   // OUR ROUTE: http://localhost:3000/contests/1/classes/1/Woof
   // so when we click our page we must GET data from database about our dog performance
   // in the future we will use more IDs than other properties
+  // const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [isDisqualifyModalOpen, setIsDisqualifyModalOpen] = useState(false);
+  const [isPenaltyModalOpen, setIsPenaltyModalOpen] = useState(false);
+  const handleDesqualification = () => {
+    console.log(`Doggo o imieniu ${ourTestDogName} wylatuje z konkursu`);
+    setIsDisqualifyModalOpen(false);
+  };
+  const handlePenalty = () => {
+    console.log(`Doggo o imieniu ${ourTestDogName} ma mnie o 10pkt`);
+    setIsPenaltyModalOpen(false);
+  };
```

Comment:
These lines will require logic on modyfing the competitors score depending on what happens.